dataactcore/config.py

```diff
@@ -77,6 +81,8 @@
     # just use the directory
     broker_files = os.path.split(broker_files)[0]
 normpath(broker_files)
+if broker_files[-1] != os.path.sep:
+    broker_files = "".join([broker_files, os.path.sep])
```

Comment:
Broker files is used when `use_aws` is set to false, that is why `os.path.sep` is needed. and I agree with the simpler concatenation. Will make that change.